apps/webapp/utils/validation.ts

```diff
@@ -0,0 +1,8 @@
+const BECH32_PREFIX = 'penumbrav2t';
+const ADDRESS_LENGTH = 146;
+
+export const validateAmount = (value: string, balance: number) =>
+  Boolean(value) && Number(value) > balance;
+
+export const validateRecepient = (value: string) =>
```

Comment:
```diff
- validateRecepient
+ validateRecipient
```

Reply:
Also, can you write tests for this.

Reply:
We should make a note for later to make use of this in the wasm crate:

```rust
impl std::str::FromStr for Address {
    type Err = anyhow::Error;

    fn from_str(s: &str) -> Result<Self, Self::Err> {
        pb::Address {
            inner: bech32str::decode(s, bech32str::address::BECH32_PREFIX, bech32str::Bech32m)?,
            alt_bech32m: String::new(),
        }
        .try_into()
    }
}
```

and then expose it to be used here. That is probably the source of truth method.